### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: fuzz/fuzz.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/python3
```
Comment (xansec) [inline fuzz/fuzz.py]: Only request here is to change the name of the file to something that describes what is being fuzzed. You would want to apply the change in the Mayhemfile target name as well.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by ebell495** [comment]
I've updated the name of the file

**Comment by xansec** [comment]
Thanks, approved!

**Pull request merged**